share/ci/run_pmacc_tests.sh

```diff
@@ -108,4 +108,7 @@ export LD_LIBRARY_PATH=/opt/boost/${BOOST_VERSION}/lib:$LD_LIBRARY_PATH
 cmake $CMAKE_ARGS $code_DIR/include/pmacc
 make
 
-ctest -V
+# runtime tests with HIP are temporary disabled due to CI issues
+if [ "${backend_cfg[0]}" != "hip" ] ; then
+  ctest -V
+fi
```

Comment:
New line missing, is it critical here?

Reply:
I guess we could let it run, and then if necessary fix and force-merge

Reply:
I fixed it because `${backend_cfg[0]}` is not available (copy past bug).